src/prognostic_equations/edmfx_entr_detr.jl

```diff
@@ -0,0 +1,284 @@
+#####
+##### EDMF entrainment detrainment
+#####
+
+"""
+   Return entrainment rate [1/s].
+
+   Inputs (everything defined on cell centers):
+   - params set with model parameters
+   - ᶜz, z_sfc, ᶜp, ᶜρ, - grid-scale height, surface height, grid-scale pressure and density
+   - buoy_flux_surface - buoyancy flux at the surface
+   - ᶜaʲ, ᶜwʲ, ᶜRHʲ, ᶜbuoyʲ - updraft area, physical vertical velocity,
+                                   relative humidity and buoyancy
+   - ᶜw⁰, ᶜRH⁰, ᶜbuoy⁰ - environment physical vertical velocity,
+                              relative humidity and buoyancy
+   - dt - timestep
+"""
+
+function pi_groups_entr(
```

Comment:
Should we just change the name to be entrainment (and then detrainment for the other one)? Since now it's pi-groups only for one of the choices?

Reply:
Yes! I will change it.